### ISSUE TITLE
Precision and recall values for classes

### ISSUE THREAD

**Issue by dinarkino** [bes-dev/mean_average_precision#19]
Thank you for the repo! The quality of the code is very good, but some things are not clear. Is it possible to get precision and recall values for each class for the specified IoU value? Also, is it possible to somehow get TP, FP values out of the metric_fn?

**Comment by bes-dev** [comment]
@dinarkino our library returns dictionary that contains AP, Precision, Recall for each class and each IoU threshold (you can find format of the output dict here: https://github.com/bes-dev/mean_average_precision/blob/develop/mean_average_precision/mean_average_precision_2d.py#L81), but we does not return TP, FP

**Comment by dinarkino** [comment]
@bes-dev If I'm not mistaken, Recall in the output is a numpy array with different Recall values for all the unique confidence thresholds in the list of detections for specified class and IoU values. Then to get Recall for predicted objects and ground truth, we should take the last value in the Recall array. That is assuming that we want to count all the preds that we added to the metric class. For example, for IoU 0.5 and class 0:
`metric_fn.value(iou_thresholds=0.5)[0]['precision'][-1]`

I wonder if I understood this correctly from the documentation? I compared obtained results with [metrics from the yolov5 package](https://github.com/ultralytics/yolov5/blob/f17c86b7f0d2038288d7292cb82dec2433cc91e5/utils/metrics.py#L117) (which is originally from https://github.com/rafaelpadilla/Object-Detection-Metrics), and I'm getting completely different results. In contrast, mAP values for each class individually and all classes together are close to each other in two packages.
